ui/redux/actions/rewards.js

```diff
@@ -185,3 +185,9 @@ export function doFetchRewardedContent() {
     Lbryio.call('reward', 'list_featured').then(success, failure);
   };
 }
+
+// @flow
+export const doUserViewRateList = () => async (dispatch: Dispatch) =>
+  await Lbryio.call('user_rewards', 'view_rate').then((data) =>
+    dispatch({ type: ACTIONS.USER_VIEW_RATE_COMPLETED, data })
+  );
```

Comment:
:small_blue_diamond: Minor:  double call if refreshed on `/$/channels`

<img width="241" alt="image" src="https://user-images.githubusercontent.com/64950861/186059621-21d6ebe5-76cb-4c40-8262-d0c0260858fe.png">


Probably don't need a full redux flag, but just a local jsx state?

Reply:
forgot a `isFetching` f38cb3fbf22ac68aad6f1c821c5da39a544214c7 this should make it 1 call only for each